agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java

```diff
@@ -17,6 +17,9 @@
 /** This class allows you to attach the Application Insights agent for Java at runtime. */
 public final class ApplicationInsights {
 
+  public static final String APPLICATIONINSIGHTS_RUNTIMEATTACH_CONFIGURATION_FILE =
+      "applicationinsights.runtimeattach.configuration.file";
```

Comment:
```suggestion
      "applicationinsights.runtime.attach.configuration.file";
```

Reply:
I was thinking "runtimeattach" one word since it represents one concept, and we would have other `applicationinsights.runtime.*` properties

However, separating the words does match what we've done with the artifact name `applicationinsights-runtime-attach`.

In OpenTelemetry Instrumentation, we'd probably use `applicationinsights.runtime-attach.configuration.file`, wdyt of that?

Reply:
runtime-attach works for me too.

Reply:
Change done with `applicationinsights.runtime-attach.configuration.file`